e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -149,6 +149,43 @@ internal static partial class TestHelper {
             await registryManager.UpdateTwinAsync(twin.DeviceId, patch, twin.ETag, ct);
         }
 
+        /// <summary>
+        /// Call rest API
+        /// </summary>
+        /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
+        /// <param name="method">REST method (Get, Post, Delete...)</param>
+        /// <param name="route">Route for the url</param>
+        /// <param name="body">Body for the request</param>
+        /// <param name="queryParameters">Additional query parameters</param>
+        public static IRestResponse CallRestApi(
+            IIoTPlatformTestContext context,
+            Method method,
+            string route,
+            object body = null,
+            Dictionary<string, string> queryParameters = null
+        ) {
+            var cancellationTokenSource = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
```

Comment:
please make it possible for tests to pass their own CancellationToken, so they can stick to their own timeout